pkg/kubernetes/scripts/virtual-machines/types.jsx

```diff
@@ -65,3 +65,7 @@ export type VmMessages = {
   message: string,
   detail: Object,
 }
+
+export type Pod = Object; // TODO
```

Comment:
The code only uses `Pod.metadata.*` properties. Does this happen to be the same type as e. g. `Vm` or real container pods, so that this could be reused instead of this dummy?

Reply:
This is the sort of `Vm` type, I will add proper definition here.